econet_heatpump_water_heater.yaml

```diff
@@ -1,12 +1,19 @@
 substitutions:
   name: "econet-hpwh"
+  friendly_name: "Water Heater"
   device_description: "Rheem Heat Pump Water Heater"
   tx_pin: GPIO19
   rx_pin: GPIO22
+  min_temperature: "43.3333"
+  max_temperature: "60"
```

Comment:
No need to have substitutions for these. Just inline them.

Reply:
Makes sense as these never change, as far as I can tell.